### DISCUSSION TITLE
Creating Home Widget - Easy And Faster

### DISCUSSION THREAD

**Discussion by bhargavraviya** [nylo-core/nylo#77] [Ideas]
- creating a home widget using the home_widget package
- make:homewidget command for fasting Create a home widget for Android and iOS native code easily.

**Comment by agordn52** [comment]
Hi @bhargavraviya,

Thanks for the idea but I think it's best for Flutter developers to use the https://pub.dev/packages/home_widget package in their project as a dependency rather than including it in the framework.

Just my thought 💭
